.github/workflows/dotnet-build.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Dotnet linter
         run: |
           dotnet tool install -g dotnet-format
-          dotnet format . --check
+          #dotnet format . --check
```

Comment:
@miazamrai can you please uncomment the linter check

Reply:
> @miazamrai can you please uncomment the linter check

Enabled again. Did it for testing.